### DISCUSSION TITLE
Recovering emails - After migrating domain to Vercel, forgot to add MX field for SMTP google

### DISCUSSION THREAD

**Discussion by doruksahin** [vercel/community#6029] [Help] [answered]
### Summary

Hello, i wish you have a good day.
I had a domain at another provider and i migrated to vercel.
however, i forgot to add MX smtp.google.com... 
I added it not and I can retrieve mails from now on. 

For 10 days i couldn't get my mails.
Is there a way to recover these emails?

### Steps to Reproduce

1. migrate domain from another provider to vercel
2. try to send mail to your mail account associated with that domain

**Comment by amyegan** [comment, marked as answer]
Hi @doruksahin. Any emails sent during the time the MX record was missing would have bounced or failed. 

https://support.google.com/a/thread/217469282/how-do-i-recover-missing-emails-caused-by-missing-mx-records?hl=en
